Backend/Lyzer/Controllers/RacesController.cs

```diff
@@ -1,28 +1,29 @@
-using Lyzer.Common.DTO;
-using Lyzer.Services;
-using Microsoft.AspNetCore.Mvc;
-
-
-namespace Lyzer.Controllers
-{
-    [ApiController]
-    [Route("/api/v1/races")]
-    public class RacesController : ControllerBase
-    {
-
-        private readonly ILogger<RacesController> _logger;
-        private readonly RacesService _racesService;
-
-        public RacesController(ILogger<RacesController> logger, RacesService racesService)
-        {
-            _logger = logger;
-            _racesService = racesService;
-        }
-
-        [HttpGet("{season}", Name = "GetSeasonRaces")]
-        public async Task<RacesDTO> GetSeasonRaces(string season)
-        {
-            return await _racesService.GetCachedRaces(season);
-        }
-    }
-}
+using Lyzer.Common.DTO;
+using Lyzer.Services;
+
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Lyzer.Controllers
+{
+    [ApiController]
+    [Route("/api/v1/races")]
+    public class RacesController : ControllerBase
+    {
+
+        private readonly ILogger<RacesController> _logger;
+        private readonly RacesService _racesService;
+
+        public RacesController(ILogger<RacesController> logger, RacesService racesService)
+        {
+            _logger = logger;
+            _racesService = racesService;
+        }
+
+        [HttpGet("current", Name = "GetSeasonRaces")]

```

Comment:
Only change I made here is, changing this from {season} to "current"
@LittleClumsy 

@Pranav-Zodes-44 could you please review

- JolpicaClient
- JolpicaConstants
- RacesDTO
- RacesService
- RacesController

(And anything else you would like to review, but those are the changes we missed with the accidental main push)